napari/components/_viewer/model.py

```diff
@@ -104,9 +105,22 @@ def cursor(self):
     def cursor(self, cursor):
         if cursor == self.cursor:
             return
-        self._qt.set_cursor(cursor)
+        self._qt.set_cursor(cursor, self.cursor_size)
         self._cursor = cursor
 
+    @property
+    def cursor_size(self):
+        """int | None: Size of cursor if custom. None is yields default size
+        """
+        return self._cursor_size
+
+    @cursor_size.setter
+    def cursor_size(self, cursor_size):
```

Comment:
maybe name this set_cursor_size, so you don't get confused with the property

Reply:
hmm not sure I quite get this - i feel like in general we have the convention of having the property and the setter specified in this way